horace_core/sqw/@sqw/private/cut_data_from_array.m

```diff
@@ -72,11 +72,11 @@
 
 % Copy data from ranges that may contribute to cut - we assume that if can hold the full data, we will have enough space to hold subset
 if hor_log_level>=1, bigtic(1), end
-v = zeros(ndatpix,npix_read);
+cut_pix_data = PixelData(zeros(ndatpix,npix_read));
```

Comment:
`PixelData()` has internal data `zeros(9, 0)` (which seemed to be the convention for an empty pix array). `PixelData(zeros(cols, rows))` has internal data with size `cols, rows` (for the time being you'll get an error if `cols ~= 9`)

Reply:
Would `PixelData(numRows)` be a simpler constructor for people to use, rather than requiring them to create their own `9xnumRows` array of zeros to initialise it??

Reply:
these calls can be updated to use the new PixelData(numRows) constructor